include/highfive/H5PropertyList.hpp

```diff
@@ -64,9 +66,6 @@ class PropertyListBase: public Object {
         return plist;
     }
 
-  protected:
-    using Object::Object;
-
```

Comment:
For my understanding: Removing these lines is just for sake of tidying up, right? Because (most) `Object::Object` are already protected, they wouldn't become public anyway.